src/redux/books/books.js

```diff
@@ -0,0 +1,24 @@
+/** @format */
+export const CREATE_BOOK = 'bookstore/books/CREATE';
+export const REMOVE_BOOK = 'bookstore/books/REMOVE';
```

Comment:
- [Optional] I think the pattern should end with the action type that's:

```
export const CREATE_BOOK = 'bookstore/books/CREATE_BOOK';
export const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
```

Reply:
Thanks for your feedback, I will modify it.